buildSrc/private/src/main/kotlin/androidx/build/AndroidXImplPlugin.kt

```diff
@@ -176,32 +170,9 @@ class AndroidXImplPlugin @Inject constructor(val componentFactory: SoftwareCompo
         @OptIn(ExperimentalKotlinGradlePluginApi::class)
         project.pluginManager.withPlugin("org.jetbrains.kotlin.multiplatform") {
             project.extensions.configure(KotlinMultiplatformExtension::class.java) {
-                val commonOrJvmPlatforms = setOf(
-                    KotlinPlatformType.common,
-                    KotlinPlatformType.jvm,
-                    KotlinPlatformType.androidJvm
-                )
-
-                /**
-                 * Here we configure the language version.
-                 * For klibs targets we can't use K1 anymore, so we switch to K2.
-                 * For k/jvm targets we still use K1 (LV 1_9).
-                 * For common metadata compilations the LV is 1_9 too, because otherwise
-                 * k/jvm compilations do not work:
-                 * `The language version of the dependent source set must be greater than or equal to that of its dependency.`
-                 * It happens because jvmMain is considered a common (metadata).
-                 */
-                it.targets.all { target ->
-                    val lv = if (target.platformType in commonOrJvmPlatforms) {
-                        org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_9
-                    } else {
-                        org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_2_1
-                    }
-
-                    (target as? HasConfigurableKotlinCompilerOptions<*>)?.compilerOptions?.languageVersion?.set(lv)
-                }
                 it.compilerOptions {
-                    apiVersion.set(org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_9)
+                    languageVersion.set(KotlinVersion.KOTLIN_2_0)
```

Comment:
We need to change Release notes in https://github.com/JetBrains/compose-multiplatform-core/pull/1778, removing "native and web "